internal/protocol/guestrequest/types.go

```diff
@@ -41,3 +41,16 @@ type RS4NetworkModifyRequest struct {
 	RequestType       RequestType `json:"RequestType,omitempty"`
 	Settings          interface{} `json:"Settings,omitempty"`
 }
+
+var (
+	// V5 GUIDs for SCSI controllers
+	// These GUIDs are created with namespace GUID "d422512d-2bf2-4752-809d-7b82b5fcb1b4"
+	// and index as names. For example, first GUID is created like this:
+	// guid.NewV5("d422512d-2bf2-4752-809d-7b82b5fcb1b4", []byte("0"))
+	ScsiControllerGuids = []string{
```

Comment:
It might be nice to add a test here that validates each of these parse properly as a GUID. Otherwise there is risk of one being added/modified that isn't really a GUID, which would break our ability to find the controller in the guest.

Reply:
Added a test.